modules/tinymce/src/core/main/ts/fmt/CaretFormat.ts

```diff
@@ -165,6 +165,29 @@ const cleanFormatNode = (editor: Editor, caretContainer: Node, formatNode: Eleme
   }
 };
 
+const normalizeNbspsBetween = (editor: Editor, caretContainer: Node | null) => {
+  const handler = () => {
+    if (caretContainer && !editor.dom.isEmpty(caretContainer)) {
+      Traverse.prevSibling(SugarElement.fromDom(caretContainer)).each((node) => {
+        if (NodeType.isText(node.dom)) {
```

Comment:
There is a sugar predicate for this `SugarNode.isText`.

Reply:
thanks, I changed them